pyspark-notebook/Dockerfile

```diff
@@ -15,11 +15,11 @@ USER root
 # Spark dependencies
 # Default values can be overridden at build time
 # (ARGS are in lower case to distinguish them from ENV)
-ARG spark_version="3.3.1"
-ARG hadoop_version="3"
+ARG spark_version="3.2.3"
+ARG hadoop_version="3.2"
 ARG scala_version
```

Comment:
A comment to clarify when a newer version can be adopted would be helpful in situations like these.

Reply:
This is a special case. I have backport a PR to branch 3.2. 
The hope here wos to have a build with spark 3.2.3 and then revers this PR back to 3.3.1